frontend/interfaces/software.ts

```diff
@@ -455,9 +455,10 @@ export interface IHostSoftwareWithUiStatus extends IHostSoftware {
 
 /**
  * Allows unified data model for rendering of host VPP software installs and uninstalls
+ * Optional as pending may not have a commandUuid
  */
 export type IVPPHostSoftware = IHostSoftware & {
-  commandUuid: string;
+  commandUuid?: string;
```

Comment:
Do we have this tracked as a BE bug yet?

Reply:
Is this a downstream affect of [this one](https://github.com/fleetdm/fleet/issues/31459)?

Reply:
> Is this a downstream affect of https://github.com/fleetdm/fleet/issues/31459?
Yes

Maybe @konstantin would know -- we're using this for pending install of a vpp as well which might not have the command_uuid /information returned on the last_install data until its installed?

Reply:
I thought even creating a pending request to install a vpp app gets an UUID assigned to it
https://github.com/fleetdm/fleet/blob/main/server/datastore/mysql/vpp.go#L886
Looks like `InsertHostVPPSoftwareInstall` requires a `commandUUID`. 
If I am understanding the context correctly, we _should_ be able to get rid of the optionality `?`.